ouroboros-network/src/Ouroboros/Network/Snocket.hs

```diff
@@ -0,0 +1,405 @@
+{-# LANGUAGE CPP                 #-}
+{-# LANGUAGE GADTs               #-}
+{-# LANGUAGE RankNTypes          #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Ouroboros.Network.Snocket
+  ( -- * Snocket Interface
+    Accept (..)
+  , AddressFamily (..)
+  , Snocket (..)
+    -- ** Socket based Snocktes
+  , SocketSnocket
+  , socketSnocket
+  , rawSocketSnocket
+    -- ** Client Snockets
+    -- Using unix sockets (posix) or named pipes (windows)
+    --
+  , LocalSnocket
+  , localSnocket
+  , LocalAddress
+  , clientAddressFromPath
```

Comment:
Shouldn't this be `localAddressFromPath`?

Reply:
yes, it should :)

Reply:
done